src/utils.ts

```diff
@@ -13,7 +13,10 @@ export function isFormField(element: Node): boolean {
   )
 }
 
-export function fireDeterminedAction(el: HTMLElement): void {
+export function fireDeterminedAction(el: HTMLElement, detail: string[]): void {
+  const delegateEvent = new CustomEvent('hotkey-fire', {cancelable: true, detail})
```

Comment:
Nit: It might be good to change `detail` to an object with a property `path` in case this event needs to include more data in the future.